src/test/java/org/jitsi/utils/collections/JMapTest.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright @ 2018 - present 8x8, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jitsi.utils.collections;
+
+import org.junit.*;
+
+import static org.jitsi.utils.collections.JMap.entry;
+import static org.junit.Assert.*;
+
+public class JMapTest
+{
+
+    @Test
+    public void testMapOf()
+    {
+        java.util.Map data = JMap.ofEntries(
```

Comment:
You don't explicitly declare the K and V of the Map here -- do we want to test the case where you have that?

Reply:
i'm not sure there's a distinction between the cases, actually.  i think the types are derived from the given entries.

Reply:
at least,  for the purpose of the test.